assets/src/components/ResourceChartNarrative.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+function ResourceChartNarrative ({ data, weekRange, gradeSelection, resourceType }) {
+  const resourceData = data.sort((a, b) => b.total_percent - a.total_percent)
```

Comment:
Do you think it makes sense to sort this data higher up in the logic, maybe after it comes back from the API? Then you don't have to repeat this sorting logic in the narrative React component AND the d3 logic.

Reply:
May be I just need to understand the over all impact from the Graph perspective, since this was done purely for the Narrative and I just repeated this when I made narrative as react component.

Reply:
Done